lms/static/js/search/base/views/search_results_view.js

```diff
@@ -14,7 +14,7 @@ define([
         // these should be defined by subclasses
         el: '',
         contentElement: '',
-        resultsTemplateId: '',
+        revisionListTemplateId: '',
```

Comment:
Oh my, not sure how I missed this. This would actually be the first PyCharm refactor fail I've ever seen, so I'm going to blame it on something I did (and not double checking the diff). Fixed. :)